Animations/Fade/UpBig.js

```diff
@@ -1,24 +1,24 @@
 var makeAnimation = require('../../src/TuxAnimations');
```

Comment:
Add 'use strict'; on line one and leave a newline between that and variable declarations.